cpp/mg_utility/mg_utils.hpp

```diff
@@ -159,9 +162,42 @@ std::unique_ptr<mg_graph::Graph<TSize>> GetGraphView(mgp_graph *memgraph_graph,
     }
   }
 
+
   return graph;
 }
 
+
+std::unordered_map<std::uint64_t,std::vector<std::uint64_t>> GetAdjacencyMatrix(mgp_graph *memgraph_graph,  mgp_memory *memory, bool weighted = false,
+                                                     const char *weight_property = nullptr,
+                                                     double default_weight = 1.0, bool directed = true){
```

Comment:
I’d put this function at the top/bottom of the namespace to make it clearer that it has nothing to do with `GraphView`.